web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -25,19 +25,38 @@ import React, {
 
 import { wait } from 'shared/utils/wait';
 
-import { Attempt, makeSuccessAttempt, useAsync } from 'shared/hooks/useAsync';
+import {
+  Attempt,
+  makeEmptyAttempt,
+  makeSuccessAttempt,
+  useAsync,
+} from 'shared/hooks/useAsync';
 
 import { RootClusterUri } from 'teleterm/ui/uri';
 import { useAppContext } from 'teleterm/ui/appContextProvider';
 
+import { Server } from 'teleterm/services/tshd/types';
+
 import type {
   AgentProcessState,
   MainProcessClient,
 } from 'teleterm/mainProcess/types';
 
+export type AgentState =
+  | AgentProcessState
+  | {
+      status: 'starting';
+    }
+  | {
+      status: 'stopping';
+    };
+
 export interface ConnectMyComputerContext {
-  state: AgentProcessState;
+  state: AgentState;
```

Comment:
Let's rename this to `agentState`, any place which uses this is going to rename it to `agentState` anyway since `state` alone is a very ambiguous term.

Reply:
Done